internal/testrunner/runners/system/servicedeployer/custom-agent-base-config.yml

```diff
@@ -0,0 +1,15 @@
+version: "2.3"
+services:
+  custom-agent:
+    image: "${ELASTIC_AGENT_IMAGE_REF}"
+    healthcheck:
+      test: "elastic-agent status"
+      retries: 180
+      interval: 1s
+    hostname: custom-agent
```

Comment:
nit: docker-custom-agent

In case that somebody uses `custom-agent` for enrolling a tru custom agent.